rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -283,12 +283,13 @@ public void downloadLicenseInfo(@PathVariable("id") String id,
         final String projectName = sw360Project.getName();
         final String projectVersion = sw360Project.getVersion();
 	final String timestamp = SW360Utils.getCreatedOnTime().replaceAll("\\s", "_").replace(":", "_");
-	final OutputFormatInfo outputFormatInfo = licenseInfoService.getOutputFormatInfoForGeneratorClass(generatorClassName);
+	String[] outputGeneratorClassNameWithVariant = generatorClassNameWithVariant.split("::");
+	final OutputFormatInfo outputFormatInfo = licenseInfoService.getOutputFormatInfoForGeneratorClass(outputGeneratorClassNameWithVariant[0]);
```

Comment:
Please remove white spaces.

Reply:
Could you rename the field generatorClassName to generatorClassNameWithVariantinside inside licenseInfoService.getLicenseInfoFile(..) method also.

Reply:
> Please remove white spaces.

Removed it

Reply:
> Could you rename the field generatorClassName to generatorClassNameWithVariant inside licenseInfoService.getLicenseInfoFile(..) method also.

Done